### PR TITLE
MCG OBC Updates

### PR DESCRIPTION
**Pull request by mid998**
Fresh eyes needed. Please review.

### CODE REVIEW

**Comment by mid998** [comment]
Don't forget to squash on merge.

File: training/modules/ocs4/pages/ocs.adoc [hunk 1]
```diff
@@ -1917,295 +1919,239 @@ Service` in the top navigation bar. This dashboard does not give you connection
...
-          args:
-            - '-c'
-            - 'set -x && s3cmd --no-check-certificate --signature-v2 --host $BUCKET_HOST:$BUCKET_PORT --host-bucket $BUCKET_HOST:$BUCKET_PORT du'
-          name: obc-test
+[...]
+----
+
+To view exactly how the application uses the information in the new *Secret* and *ConfigMap* have a look at the file photo-album/app.yaml after you have deployed the app. In the deployment template specification section, find 'env:' and you can see how the *ConfigMap* and *Secret* details are mapped to environment variables. 
+
+----
```
Comment (mulbc) [inline training/modules/ocs4/pages/ocs.adoc:2097]: Please mark as `[source,yaml]` so that it gets highlighting

**Review by mulbc — CHANGES_REQUESTED** [review]
lgtm, please add the yaml tag to that one source block

File: training/modules/ocs4/pages/ocs.adoc [hunk 2]
```diff
@@ -1774,19 +1774,21 @@ You can validate that Prometheus and AlertManager are working correctly after
 moving to persistent storage <<Monitoring the OCS environment>> in a later
 section of this lab guide.
 
-== Using the Multi-Cloud-Gateway
+== Create a new OCP application deployment using the Multicloud Object Gateway OBC
```
Comment (jeanchlopez) [inline training/modules/ocs4/pages/ocs.adoc:1777]: Can we shorten the title? Might wrap on smaller screens may be

File: training/modules/ocs4/pages/ocs.adoc [hunk 3]
```diff
@@ -1774,19 +1774,21 @@ You can validate that Prometheus and AlertManager are working correctly after
...
+The MCG console is not fully integrated with the *Openshift Web Console* and resources created in the MCG console
+ are not synchronized back to the Openshift Cluster. For checking MCG/NooBaa status and object bucket claims, the `NooBaa client` or `oc CLI` will be used. 
 
-NOTE: While the NooBaa Web Management Console is accessible, it should not be
-used to create any resources, since they are currently not synchronized back
-to the Openshift cluster.
+For other MCG features such as Namespace buckets, please use the MCG console.
 
-=== Checking on the MCG status
+=== Checking on MCG status
```
Comment (jeanchlopez) [inline training/modules/ocs4/pages/ocs.adoc:1787]: Simpler
--> Checking MCG status

File: training/modules/ocs4/pages/ocs.adoc [hunk 4]
```diff
@@ -1917,295 +1919,239 @@ Service` in the top navigation bar. This dashboard does not give you connection
...
 about the usage of your S3 backend as well as a link to the MCG console.
 
-=== Creating an Object Bucket Claim
+=== Creating and Using Object Bucket Claims
 
-An Object Bucket Claim, *OBC*, can be used to request a S3 compatible bucket
-backend for your workloads. When creating an OBC you get a ConfigMap (CM) and
-a Secret that together contain all the information your application needs to
-use the object storage service.
+MCG Object Bucket Claims, or *OBC*, are used to dynamically create S3 compatible buckets that can be used by
```
Comment (jeanchlopez) [inline training/modules/ocs4/pages/ocs.adoc:1924]: *OBC* is singular while "Object Bucket Claims" is plural.

File: training/modules/ocs4/pages/ocs.adoc [hunk 5]
```diff
@@ -1917,295 +1919,239 @@ Service` in the top navigation bar. This dashboard does not give you connection
...
+NAME                   READY   STATUS      RESTARTS   AGE
+photo-album-1-build    0/1     Completed   0          10m
+photo-album-1-deploy   0/1     Completed   0          10m
+photo-album-1-rtplt    1/1     Running     0          10m
 ----
 
-Inside of your `openshift-storage` namespace, you will now find the
-*ConfigMap* and the *Secret* to use this OBC. The CM and the secret have the
-same name as the *OBC*:
+Now that the photo-album application has deployed you can view the *Object Bucket Claim* it created. You can also view the new bucket, *Configmap* and *Secret* using the oc command.
```
Comment (jeanchlopez) [inline training/modules/ocs4/pages/ocs.adoc:2009]: application has deployed -> application has been deployed

File: training/modules/ocs4/pages/ocs.adoc [hunk 6]
```diff
@@ -1917,295 +1919,239 @@ Service` in the top navigation bar. This dashboard does not give you connection
...
-.Example output:
-----
-NAME             READY   STATUS      RESTARTS   AGE
-obc-test-bvg8h   0/1     Completed   0          22s
-----
+image::photo-album-select-upload.png[]
 
-Then you can check the `obc-test` *Pod* logs for the contents of the S3 bucket
-using the command below (in this case there are zero objects in the bucket).
+Select 1 or more photos, then make sure to upload.
```
Comment (jeanchlopez) [inline training/modules/ocs4/pages/ocs.adoc:2140]: make sure to upload -> make sure to click the upload button

**Review by jeanchlopez — COMMENTED** [review]
Some ideas to consider. General flow is great.

**Review by mulbc — APPROVED** [review]

**Pull request merged**